test/k8s-integration/main.go

```diff
@@ -602,3 +597,32 @@ func deleteImage(stagingImage, stagingVersion string) error {
 	}
 	return nil
 }
+
+func getTempDir() string {
+	dir, err := ioutil.TempDir("", "")
```

Comment:
can we prefix or store all these tmp dirs in a way that identifies this as from this test framework. like
`gcp-pd-driver-tmp` or something. Not sure what a canonical name will look like